src/core/src/op/subtract.cpp

```diff
@@ -63,6 +63,21 @@ bool Subtract::evaluate(TensorVector& outputs, const TensorVector& inputs) const
                                       get_autob());
 }
 
+bool Subtract::evaluate_symbol(ov::TensorSymbolVector& output_symbols) const {
+    const auto &lhs_pshape = input(0).get_tensor().get_partial_shape(),
+               rhs_pshape = input(1).get_tensor().get_partial_shape();
```

Comment:
```suggestion
               &rhs_pshape = input(1).get_tensor().get_partial_shape();
```
?
currently the `rhs_pshape` is not a reference.